tasks/main.yml

```diff
@@ -11,25 +11,29 @@
   when: docker_version is not defined
 
 # https://docs.docker.com/installation/ubuntulinux/
-- name: Install trusty kernel onto 12.04
+- name: Install backported kernel on pre-16.04 LTS
```

Comment:
The kernel, from one LTS release to the previous: https://wiki.ubuntu.com/Kernel/LTSEnablementStack